spec/dbcon_spec.rb

```diff
@@ -0,0 +1,27 @@
+require_relative '../lib/dbcon'
+
+describe DatabaseConnection do
+  subject { described_class }
+
+  describe "#setup" do
+    it "sets up a connection to a database" do
+      expect(PG).to receive(:connect).with(dbname: 'chitter_test')
+      subject.setup('chitter_test')
+    end
+  end
+
+  describe ".conn" do
+    it "is an instance of PG" do
+      connection = subject.setup('chitter_test')
+      expect(subject.conn).to eq connection
+    end
+  end
```

Comment:
I like these small steps linked to the database connection. I can see where your testing has led you to code.